README.md

```diff
@@ -1,2 +1,4 @@
 # hello-world
 First project
+
```

Comment:
Why this stupid empty line?

Reply:
Because now I commited 2 lines of code. More lines more work. Who cares about quality?